changelog.rst

```diff
@@ -18,6 +18,7 @@ Added
 Changed
 +++++++
   - Updated GSD reader to use the GSD v2.0.0 API.
+  - Changed behavior of ``types``, ``typeid``, ``type_shapes`` to match HOOMD conventions.
```

Comment:
Add deprecation note for the `shapedef` property. Maybe also mention how it works from now on until its removal.